ykman/cli/fido.py

```diff
@@ -74,6 +75,7 @@ def info(ctx):
     """
     Display status of FIDO2 application.
     """
+    ctx.obj['dev'].get()
     controller = ctx.obj['controller']
```

Comment:
Related to the previous point - the relationship between this and the previous line isn't obvious. Maybe we could express this as a method on the `_DeviceResolver` class instead?

Maybe `ctx.obj['dev'].set_controller(...)` instead of `ctx.obj['controller'] = ...`?

Or maybe `ctx.obj['dev'].get_fido_controller()`, `.get_otp_controller()` etc., which call `.get()` internally?